rules/windows/builtin/win_root_certificate_installed.yml

```diff
@@ -0,0 +1,38 @@
+action: global
+title: Root Certificate Installed
+id: 42821614-9264-4761-acfc-5772c3286f76
+status: experimental
+description: Adversaries may install a root certificate on a compromised system to avoid warnings when connecting to adversary controlled web servers.
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1553.004/T1553.004.md
+author: 'oscd.community, @redcanary, Zach Stanford @svch0st'
+date: 2020/10/10
+tags:
+    - attack.defense_evasion
+    - attack.t1553.004
+level: medium
+falsepositives:
+    - Help Desk or IT may need to manually add a corporate Root CA on occasion. Need to test if GPO push doesn't trigger FP
+detection:
+    condition: 1 of them
+---
+logsource:
+    product: windows
+    service: powershell
+detection:
+    selection:
+        EventID: 4104
+        ScriptBlockText|contains:
+          - 'Import-Certificate * Cert:\LocalMachine\Root'
+          - 'Move-Item * Cert:\LocalMachine\Root'
```

Comment:
Hello @svch0stz! 

Thank you for your contribution!

Could you please change this part to something like that:

```
        - ScriptBlockText|contains|all:
          - 'Import-Certificate'
          - 'Cert:\LocalMachine\Root'
        - ScriptBlockText|contains|all:
          - 'Move-Item'
          - 'Cert:\LocalMachine\Root'
```

The thing is that some LM/SIEM systems don't process '`*`' as Splunk or Elasticsearch, so it won't work out of the box for them, and it would be hard for a SIEM expert to understand how exactly he/she supposes to construct a correlation rule looking at that detection logic.

Thank you!